packages/vue/test-app/package.json

```diff
@@ -13,16 +14,17 @@
     "sync": "sh ./scripts/sync.sh"
   },
   "dependencies": {
-    "@ionic/vue": "5.6.3",
-    "@ionic/vue-router": "5.6.3",
+    "@ionic/vue": "5.9.3",
```

Comment:
We will want to rebase against main so these target v6.

Reply:
The test app always uses the scripts to run the tests so it will move whatever the parent vue/vue-router have. You want me to have the test-app reference 6.0.2?